CHANGELOG.md

```diff
@@ -1,151 +1,228 @@
-# Release 12.1.0
+# Release 12.2.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 12.1.0_
+_This changelog only contains release notes for PWA Studio and Venia 12.2.0_
 _For older release notes, see_ [PWA Studio releases][].
 
-## New Features
+## Highlights
 
--  **Added PWA Studio metapackages** — In this release, we introduce our [PWA Studio metapackages][]: One for [Magento Open Source][] and one for [Adobe Commerce][]. These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We extended the GraphQL schema to include a new mutation and query as well as new fields that provide more details when a cart item error occurs.
+-  **New Venia CMS pages** - Venia now ships with PWA versions of the Commerce and Open Source default CMS pages: _About Us_, _Contact Us_, _Customer Service_, _Newsletter Subscription_. Out of the box, these pages are ready to use or customize as needed.
 
-    **UPDATES REQUIRED!** This release and all future PWA Studio releases will have dependencies on these metapackages. That means you need to add one or both of the metapackages to your PWA apps (depending on your backend target). Instructions for installing these packages are in the README files for each metapackage repo:
+-  **Luma and Venia can now be deployed to the same cloud environment** — This deployment option makes it easier for customers to deploy their PWA site alongside their existing Luma site while they migrate from Luma to Venia.
 
-    **For Open Source backends**: Install the [PWA Magento Open Source metapackage][].
+-  **Deprecated GraphQL fields removed from Venia** — In this release, we removed the usage of all deprecated GraphQL fields in Venia.
 
-    **For Adobe Commerce backends**: Install the [PWA Adobe Commerce metapackage][].
+-  **Venia Homepage is now the same for Adobe Commerce and Open Source** — Previously, the Venia Homepage rendered Page Builder content for Adobe Commerce sites and different content for Magento Open Source sites. But now that Page Builder is a part of Magento Open Source, the Venia Homepage renders the same content for both sites.
```

Comment:
Just noting some consistency issues - sometimes we use "Open Source" and sometimes "Magento Open Source".